llvm/include/llvm/Transforms/Vectorize/SandboxVectorizer/DependencyGraph.h

```diff
@@ -159,8 +168,37 @@ class DependencyGraph {
   /// The DAG spans across all instructions in this interval.
   Interval<Instruction> DAGInterval;
 
+  std::unique_ptr<BatchAAResults> BatchAA;
+
+  enum class DependencyType {
+    RAW,   ///> Read After Write
```

Comment:
read after write is more annoying to process than write before read, can we use `WBR` (or something similar) as opposed to `RAW`?

also maybe we should spell these out instead of using abbreviations? I mostly ask for consistency with other enum classes that are camel case instead of all uppercase (e.g. `None` instead of `NONE`)

Reply:
The reason I am using RAW and not WBR is because it's standard dependency terminology. I agree that WBR is easier to process, but let's stick to what's expected. 

Yeah, these uppercase ones need to go. I will update them.